packages/twenty-server/src/engine/metadata-modules/object-metadata/hooks/before-update-one-object.hook.ts

```diff
@@ -11,9 +12,27 @@ import {
 } from '@ptc-org/nestjs-query-graphql';
 import { Equal, In, Repository } from 'typeorm';
 
+import { WorkspaceEntityMetadataArgs } from 'src/engine/twenty-orm/interfaces/workspace-entity-metadata-args.interface';
+
 import { FieldMetadataEntity } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
 import { UpdateObjectPayload } from 'src/engine/metadata-modules/object-metadata/dtos/update-object.input';
 import { ObjectMetadataService } from 'src/engine/metadata-modules/object-metadata/object-metadata.service';
+import { metadataArgsStorage } from 'src/engine/twenty-orm/storage/metadata-args.storage';
+import { standardObjectMetadataDefinitions } from 'src/engine/workspace-manager/workspace-sync-metadata/standard-objects';
+
+function getStandardObjectMetadata(
```

Comment:
Naming for function is a bit too generic. Also usually we would move this to a dedicated utils (so make sure it's properly named so that people don't use this when the want `getTargetObjectMetadataOrThrow` or another metadata getter that reflects the current schema (more frequent use-case)

Reply:
I have renamed the function to `getEntityMetadataByStandardId`, is it ok?

Reply:
In most cases, when you're talking about an object's metadata you're talking about the database value. Here you're looking at something different which happens much more rarely which is looking at the original definition. So I would have called it something like `getStandardObjectMetadataFromDefinition`